### PR TITLE
docs: reference terraform & docs.

### PR DESCRIPTION
**Pull request by dayne**
While reading README realized I wanted reference to terraform & the github provider docs handy.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # tf-github
 
-This repo manages the ACEP DevOps Organization
+This repo manages the ACEP DevOps Organization using [terraform](terraform.io)'s [github provider integration]https://registry.terraform.io/providers/integrations/github/latest/docs)
```
Comment (dgpalmieri) [inline README.md:3]: Needs an opening parenthesis on the last link.

**Review by dgpalmieri — CHANGES_REQUESTED** [review]

**Comment by dgpalmieri** [comment]
Updated to pass style checks

**Review by TechSolomon — CHANGES_REQUESTED** [review]
Link to https://www.terraform.io/ is broken?

**Comment by dgpalmieri** [comment]
weird, the link worked fine for me. Probably good to have the rest of the window dressing though

**Review by dgpalmieri — APPROVED** [review]
who knew that this pr would take so long

**Review by TechSolomon — APPROVED** [review]
:shipit:

**Pull request merged**